doc/specs/collages.md

```diff
@@ -0,0 +1,140 @@
+# Collages
+
+## Table of contects
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+The collage system is an important feature.
+It provides an orthogonal way to either artist page or tagging system to organise Album, based (mostly) on Facts.
+
+## Business rules
+
+* Collage contains albums
+* Collage should be suscribable
+* Collage system should only permit the creation of a collage if:
+  * the user rank is higher enough,
```

Comment:
There are no 'ranks', only classes. Classes should not be ordered. "The user is allowed to create collages"